### PR TITLE
Deployment #1

### PR DESCRIPTION
**Pull request by adrienfort**
# Description

Update guiding documents, resolve security dependencies issues and add a multi-stage dockerfile :rocket:

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [ ] I have linked this PR to an issue
- [ ] I have linked this PR to a milestone
- [x] I have linked this PR to a project
- [ ] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [x] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Review by RezaRahemtola — APPROVED** [review]
LGTM :rocket:

**Comment by RezaRahemtola** [comment]
Looks like there's a dependency problem in the tests.

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
+###
+# Builder image
+###
+FROM node:16.5-alpine AS builder
 
-# Retrieve sources
 WORKDIR /app
-COPY . /app
+
+# Install dependecies
```
Comment (RezaRahemtola) [inline Dockerfile:8]: small typo (`dependencies`)

**Review by Samoten777 — COMMENTED** [review]
LGTM

**Pull request merged**